student/src/components/Pages/Home/index.js

```diff
@@ -37,7 +68,6 @@ class Home extends Component {
     };
 
   render() {
-      console.log("State open:", this.state.open);
     return (
       <React.Fragment>
         <StyledContent>
```

Comment:
Just add conditional rendering statment to render "Loading..." unless the students state is 2

Reply:
done :+1: